homebrew_challenge/src/components/HeatMap/Heatmap.jsx

```diff
@@ -4,12 +4,14 @@ import LoadingComponent from "../LoadingComponent/LoadingComponent";
 import {
   AREATYPE_COUNCIL_AREAS,
   VALUETYPE_DEATHS,
+  VALUETYPE_CASES
```

Comment:
Turns out you're not using this after all...

Reply:
removed